pyteal/ir/tealblock.py

```diff
@@ -14,9 +14,10 @@
 class TealBlock(ABC):
     """Represents a basic block of TealComponents in a graph."""
 
-    def __init__(self, ops: List[TealOp]) -> None:
+    def __init__(self, ops: List[TealOp], root_expr: Optional["Expr"] = None) -> None:
```

Comment:
if you propose `... | None`, then consider the change
```suggestion
    def __init__(self, ops: List[TealOp], root_expr: "Expr | None" = None) -> None:
```

Reply:
Do you think it's worthwhile to have a separate PR that goes in front of this one, and just removes all `Optional`'s in files touched by this one? That would reduce the diffs a bit, as opposed to me fixing it everywhere. Unfortunately, I've already been inconsistent here, sometimes not fixing, sometimes not, and in one case, being _forced_ to fix by `ruff` when I was editing a file inside of **AlgoKit**.

Reply:
I've made the change in 56041f0a971abf91c3663deb52c46af91d5166cb